### DISCUSSION TITLE
Choosing unifi network application  version

### DISCUSSION THREAD

**Discussion by poseidogithub** [tteck/Proxmox#3637] [Help Needed]
Hello!

I'm new to Proxmox, and after discovering @tteck's scripts, I figured I'd give them a shot—they're supposed to be easy enough for beginners like me!

I'm currently experimenting with LXC containers, and one of the containers I want to set up is the UniFi Network Server. However, I need a specific version (8.1.113) because it's the last one that supports meshing for older APs (learned that the hard way). I thought I'd take a look at the script and just modify the version there, but I can't seem to figure out how to do it. I don't see where it downloads the version: https://[raw.githubusercontent.com/tteck/Proxmox/main/ct/unifi.sh]. I was expecting it to be more like Docker, where you can simply select the version you want to download.

Any help would be appreciated!

Thanks!

**Comment by tteck** [comment]
The script uses UniFi's APT repository, so I don't think there's a way to control the version.
https://github.com/tteck/Proxmox/blob/e680dab28fd506e8ad0624275035c4fcb7ec48e8/install/unifi-install.sh#L38-L43
